src/coder/encoder.rs

```diff
@@ -466,6 +466,54 @@ impl Encoder {
     pub fn set_bandwidth(&mut self, bandwidth: Bandwidth) -> Result<()> {
         self.set_encoder_ctl_request(ffi::OPUS_SET_BANDWIDTH_REQUEST, bandwidth as i32)
     }
+
+    /// Gets encoder's configured use of discontinuous transmission.
+    pub fn dtx(&self) -> Result<bool> {
+        self.encoder_ctl_request(ffi::OPUS_GET_DTX_REQUEST)
+            .map(|n| n == 1)
+    }
+
+    /// Configures the encoder's use of discontinuous transmission (DTX).
+    pub fn set_dtx(&mut self, dtx: bool) -> Result<()> {
+        let dtx_shall_be_enabled = if dtx {1} else {0};
+
+        self.set_encoder_ctl_request(ffi::OPUS_SET_DTX_REQUEST, dtx_shall_be_enabled)
+            .map(|_| ())
+    }
+
+    /// Enables the encoder's use of discontinuous transmission (DTX).
+    pub fn enable_dtx(&mut self) -> Result<()> {
+        self.set_dtx(true)
+    }
+
+    /// Disables the encoder's use of discontinuous transmission (DTX).
+    pub fn disable_dtx(&mut self) -> Result<()> {
+        self.set_dtx(false)
+    }
+
+    /// Gets the encoder's configured signal depth.
+    pub fn lsb_depth(&self) -> Result<u8> {
+        self.encoder_ctl_request(ffi::OPUS_GET_LSB_DEPTH_REQUEST)
+            .map(|n| n as u8)
+    }
+
+    /// Configures the depth of signal being encoded.
+    ///
+    /// This is a hint which helps the encoder identify silence and near-silence.
+    /// It represents the number of significant bits of linear intensity below
+    /// which the signal contains ignorable quantization or other noise.
```

Comment:
```suggestion
    /// which the signal contains ignorable quantisation or other noise.
```
Please use British English.